externs/olx.js

```diff
@@ -5567,31 +4943,35 @@ olx.source.StaticVectorOptions.prototype.object;
  * @type {ol.proj.ProjectionLike}
  * @api
  */
-olx.source.StaticVectorOptions.prototype.projection;
+olx.source.VectorOptions.prototype.projection;
 
 
 /**
- * Text.
- * @type {string|undefined}
+ * State.
+ * @type {ol.source.State|string|undefined}
  * @api
  */
-olx.source.StaticVectorOptions.prototype.text;
+olx.source.VectorOptions.prototype.state;
 
 
 /**
- * URL.
- * @type {string|undefined}
+ * The loading strategy to use. By default an {@link ol.loadingstrategy.all}
+ * strategy is used, which means that features at loaded all at once, and
+ * once.
+ * @type {function(ol.Extent, number): Array.<ol.Extent>|undefined}
```

Comment:
Would it make sense to introduce a type for the strategy function, similar to the loader function?